Go/Go.sublime-syntax

```diff
@@ -114,30 +117,76 @@ contexts:
 
   # https://golang.org/ref/spec#Comments
   match-comments:
-    # Go has magic comments in the form of:
+   # Go has magic comments in the form of:
```

Comment:
Everything else seems fine to me.

Reply:
Nice catch, this should be fixed before merge. Workflow should've caught this, but it didn't auto-run because this is the first contribution to the repo.

Reply:
That was editor fumbling. Will fix.

Reply:
Done